guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -12,6 +12,12 @@ There are no highlights with Foreman {ProjectVersion}.
 // If this section would be empty otherwise, uncomment the following line:
 There are no upgrade warnings with Foreman {ProjectVersion}.
```

Comment:
I'm assuming the added heading is an upgrade warning.
```suggestion
// There are no upgrade warnings with Foreman {ProjectVersion}.
```

Reply:
I don't consider this a warning. IMO it's "just" a normal migration.